### PR TITLE
Bug/ser 219 yarn jenkins

### PR DESCRIPTION
**Pull request by rmharrison**
#### What's this PR do?
Separates `yarn test` and `yarn jenkins`.
No new functionality.

NB: There are also no release tags on survey. After this branch is merged down, I'd intend to create a release.

#### Related JIRA tickets:
https://jira.amida.com/browse/SER-219

#### How should this be manually tested?
`yarn jenkins` (NB: Survey has 2k tests and takes 5 min to run)
 
#### Screenshots (if appropriate):

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,20 @@ A list of full environment variable settings is below.  They can be either manua
 Multitenancy is supported through postgres schemas.  Multiple schemas are specified using `SURVEY_SERVICE_DB_SCHEMA` as a '~' delimited string of schema names.  This project assumes that each schema has the same table structure during database synchronization.  Schema names are appended to the base url for each API end point so that each tenant can be accessed using a different path.
 
 ## Tests
-`yarn test`
+```sh
+# Make sure .env.test exists
```
Comment (unprofessional) [inline README.md:123]: Do we want to include a `.env.test` and/or `.env.production` in the code base as is in the Messaging Service? Or do we want to remove the ones in the Messaging Service? If the latter, do we want to either introduce a `.env.test.example` of, if there's no difference here other than a change in the `NODE_ENV`, then perhaps make a note here in the README to create `.env.test` and/or `.env.production` when copying for the regular `.env`?

**Review by unprofessional — CHANGES_REQUESTED** [review]
Moreover a documentation question but also pertains to https://github.com/amida-tech/amida-messaging-microservice/pull/86

**Comment by unprofessional** [comment]
Some notes.
- All tests pass with a `yarn test`
- Running `docker-compose -f docker-compose.yml up` runs into a problem whereby it complains:
  - Is there a way to have the Dockerfile/compose to create the DB such that the compose doesn't fail in this manner? Either that, or I'm just doing something wrong~
```
postgres_1   | FATAL:  database "amida_survey_microservice" does not exist
amida-api_1  | Server failed to start due to error: SequelizeConnectionError: database "amida_survey_microservice" does not exist
amida-api_1  | Done in 8.90s.
amida-survey-microservice_amida-api_1 exited with code 0
```
- Out of scope for this ticket -- but do we want to upgrade it to pg 9.6 like with the Messaging Service?

**Review by nickmccally — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rmharrison) [reply README.md:123]: @unprofessional : You're correct that `.env.test` should be included. Included in a separate PR, https://github.com/amida-tech/amida-survey-microservice/pull/71

**Comment by rmharrison** [comment]
> Out of scope for this ticket -- but do we want to upgrade it to pg 9.6 like with the Messaging Service?
@unprofessional : Good catch. Fixed in PR #71 

> docker-compose complaining
Punting. Noted in [SER-251](https://jira.amida.com/browse/SER-251).

**Pull request merged**